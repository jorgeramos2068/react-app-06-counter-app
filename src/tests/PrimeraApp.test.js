import React from 'react';
import { shallow } from 'enzyme';
import PrimeraApp from '../PrimeraApp';

describe('Test en <PrimeraApp />', () =>{
  test('Debe de mostrar <PrimeraApp /> correctamente', () => {
    const saludo = 'Hola Superman';
    const wrapper = shallow(<PrimeraApp saludo={saludo} />);
    expect(wrapper).toMatchSnapshot();
  });

  test('Debe de mostrar el subtÃ­tulo enviado por props', () => {
    const saludo = 'Hola Superman';
    const subtitulo = 'Nuevo subtitulo';
    const wrapper = shallow(<PrimeraApp saludo={saludo} subtitulo={subtitulo} />);
    const textoParrafo = wrapper.find('p').text();
    expect(textoParrafo).toBe(subtitulo);
  });
});
