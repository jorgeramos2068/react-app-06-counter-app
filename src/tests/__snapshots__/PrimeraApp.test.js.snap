// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en <PrimeraApp /> Debe de mostrar <PrimeraApp /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola Superman
  </h1>
  <p>
    Soy un subt√≠tulo
  </p>
</Fragment>
`;
